# HG changeset patch
# User ceisserer
# Date 1217704459 -7200
# Node ID f757686901c7e70f76bfa4860ea2e0c16d7e7252
# Parent  4b11af86e814b7d62e9e1e19788d527987bb209c
Summary: Added some missing mappings which caused failures in non-fastdebug mode

diff -r 4b11af86e814 -r f757686901c7 make/sun/awt/mapfile-mawt-vers
--- openjdk/jdk/make/sun/awt/mapfile-mawt-vers	Fri Aug 01 22:43:51 2008 +0200
+++ openjdk/jdk/make/sun/awt/mapfile-mawt-vers	Sat Aug 02 21:14:19 2008 +0200
@@ -417,6 +417,36 @@
                 Java_sun_java2d_x11_X11SurfaceData_XSetForeground;
                 Java_sun_java2d_x11_X11SurfaceData_XSetGraphicsExposures;
 
+
+		Java_sun_java2d_xr_XRSurfaceData_flushNativeSurface;
+		Java_sun_java2d_xr_XRSurfaceData_XRIsDrawableValid;
+		Java_sun_java2d_xr_XRSurfaceData_setInvalid;
+		Java_sun_java2d_xr_XRSurfaceData_XRInitXRender;
+		Java_sun_java2d_xr_XRSurfaceData_initOps;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetForeground;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetComposite;
+		Java_sun_java2d_xr_XRSurfaceData_initIDs;
+		Java_sun_java2d_xr_XRSurfaceData_XRInitSurface;
+		Java_sun_java2d_xr_XRSurfaceData_XRResetClip;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetClip;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetTransformNative;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetFilter;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetTexturePaint;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetGradientPaint;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetLinearGradientPaint;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetRadialGradientPaint;
+		Java_sun_java2d_xr_XRSurfaceData_XRResetPaint;
+		Java_sun_java2d_xr_XRSurfaceData_XRSetRepeat;
+		Java_sun_java2d_xr_XRPMBlitLoops_nativeRenderBlit;
+		Java_sun_java2d_xr_XRRenderer_XRFillRect;
+		Java_sun_java2d_xr_XRRenderer_XRFillSpans;
+		Java_sun_java2d_xr_XRRenderer_XRDoPath;
+		Java_sun_java2d_xr_XRRenderer_XRDrawLine;
+        	Java_sun_java2d_xr_XRRenderer_devCopyArea;
+		Java_sun_java2d_xr_XRMaskFill_maskFill;
+		Java_sun_java2d_xr_XRMaskBlit_maskBlit;
+		XRT_DrawGlyphList;
+
                 Java_sun_java2d_opengl_OGLMaskFill_maskFill;
                 Java_sun_java2d_opengl_OGLRenderer_drawPoly;
                 Java_sun_java2d_opengl_OGLRenderQueue_flushBuffer;
diff -r 4b11af86e814 -r f757686901c7 make/sun/headless/mapfile-vers
--- openjdk/jdk/make/sun/headless/mapfile-vers	Fri Aug 01 22:43:51 2008 +0200
+++ openjdk/jdk/make/sun/headless/mapfile-vers	Sat Aug 02 21:14:19 2008 +0200
@@ -57,6 +57,8 @@
                 Java_sun_java2d_x11_X11SurfaceData_XSetXorMode;
                 Java_sun_java2d_x11_X11SurfaceData_XSetForeground;
 
+		XRT_DrawGlyphList;
+
 		X11SurfaceData_GetOps;
 		Java_java_awt_Font_initIDs;
                 Java_sun_font_FontManager_getFontConfig;
diff -r 4b11af86e814 -r f757686901c7 make/sun/xawt/mapfile-vers
--- openjdk/jdk/make/sun/xawt/mapfile-vers	Fri Aug 01 22:43:51 2008 +0200
+++ openjdk/jdk/make/sun/xawt/mapfile-vers	Sat Aug 02 21:14:19 2008 +0200
@@ -356,6 +356,7 @@
         Java_sun_java2d_xr_XRSurfaceData_XRSetComposite;
         Java_sun_java2d_xr_XRSurfaceData_initIDs;
         Java_sun_java2d_xr_XRSurfaceData_XRInitSurface;
+	Java_sun_java2d_xr_XRSurfaceData_XRSetClip;
 	Java_sun_java2d_xr_XRSurfaceData_XRResetClip;
 	Java_sun_java2d_xr_XRSurfaceData_XRSetTransformNative;
         Java_sun_java2d_xr_XRSurfaceData_XRSetFilter;
@@ -370,6 +371,7 @@
 	Java_sun_java2d_xr_XRRenderer_XRFillSpans;
         Java_sun_java2d_xr_XRRenderer_XRDoPath;
 	Java_sun_java2d_xr_XRRenderer_XRDrawLine;
+        Java_sun_java2d_xr_XRRenderer_devCopyArea;
         Java_sun_java2d_xr_XRMaskFill_maskFill;
 	Java_sun_java2d_xr_XRMaskBlit_maskBlit;
 	XRT_DrawGlyphList;
